machine.md

```diff
@@ -0,0 +1,217 @@
+# Virtualized Hardware Configuration
+
+Finetuning different aspects of the hardware which are not device related
+(BIOS, mainboard, ...) is sometimes necessary to allow Guest Operating Systems
+to properly boot and reboot.
+
+## SMBIOS Firmware
+
+In order to provide a consistent view on the virtualized hardware for the Guest
+OS, the SMBIOS UUID can be set to a constant value via `spec.firmware.uid`:
+
+```
+metadata:
+  name: myvm
+spec:
+  domain:
+    firmware:
```

Comment:
Didn't we say that `firmware` was to generic, and we wanted to go with:

```yaml
Domain:
  Loader:
    Firmware: uefi
    UUID: nnn-nnn-nnnn.....
# OR
  Bootloader:
    Smbios:
      UUID: nnnn-nnnn-nnnn-nnnnn
    Uefi|bios:
      Firmware: some reference to a blob
```

Reply:
yes, but it is not yet implemented. Just documenting what is there.

Reply:
Ack